fulp_modules/main_features/bloodsuckers/code/bloodsucker/bloodsucker_crypt.dm

```diff
@@ -58,6 +58,7 @@
  */
 
 /obj/structure/bloodsucker
+	///Who owns this structure?
```

Comment:
Ever since your mindshield PR I noticed you keep phrasing comments as questions and my hatred for you has doubled in power

Reply:
i dont like this, whos he asking

Reply:
> 
> 
> i dont like this, whos he asking

You, now answer me already shitter

Reply:
> > i dont like this, whos he asking
> 
> You, now answer me already shitter

var/mob/living/owner